src/NzbDrone.Core/Configuration/ConfigFileProvider.cs

```diff
@@ -230,6 +231,8 @@ public string InstanceName
             }
         }
 
+        public string ApplicationUrl => GetValue("ApplicationUrl", "", persist: false);
```

Comment:
This setting should live in the DB instead of the config file as it's not required for startup.

Reply:
Done.